cg/meta/demultiplex/demux_post_processing.py

```diff
@@ -61,6 +71,52 @@ def transfer_flow_cell(
 
         LOG.info(f"Flow cell added: {flow_cell}")
 
+    def add_sample_lane_sequencing_metrics_for_flow_cell(self):
+        """Add sample lane sequencing metrics to status database."""
+        flow_cell_dir: Path = self.flow_cell_dir
```

Comment:
It would not work, however the CLI command it will be used in has a required flowcell name parameter. Currently it is "optional" because the old demux code is written without having a flow cell name paramter. 

So once we deprecate the "old" functions we will make this parameter non-optional.

Reply:
It does not, but the function is never called when it has not been set. I think we should remove these attributes because it is hacky